### PR TITLE
feat: Update to latest contracts-v2 release

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import * as dotenv from "dotenv";
+
+import { HardhatUserConfig } from "hardhat/config";
```
Comment (nicholaspai) [inline hardhat.config.ts:3]: I had trouble importing the `hardhat.config` from the `contracts-v2/dist` folder as it couldn't import the files imported by that file. So I copied it directly into here and removed all nonessential parts

File: src/contracts/acrossConfigStore.test.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { Client } from "./acrossConfigStore";
+jest.useFakeTimers();
```
Comment (nicholaspai) [inline src/contracts/acrossConfigStore.test.ts:2]: These fix some warnings described by this issue: https://stackoverflow.com/questions/50793885/referenceerror-you-are-trying-to-import-a-file-after-the-jest-environment-has

File: src/constants.ts
Comment (james-a-morris) [inline src/constants.ts:1]: This is a good change +1

**Review by james-a-morris — APPROVED** [review]
LGTM - I don't see anything that sticks out to me. Otherwise, this seems like a good refactor to upgrade `contracts-v2` to a stable version

**Pull request merged**